packages/scene-composer/src/hooks/useEditorHelper.ts

```diff
@@ -53,6 +53,7 @@ export function useEditorHelper<T>(
     }
   });
 
+  /* istanbul ignore next */
```

Comment:
Logic in `useFrame` is tightly coupled to canvas rendering & should go through either manual or integration tests rather than unit tests.

Reply:
Disagree, this logic is pretty easy to mock out and test, I don't think we should ignore it. Wheras an integration test in this place has no access to the logic, and this is typically tied to animations.

Really I question whether we need this use frame at all. We need to evaluate whether something should be visible or not 60 times a second?

Reply:
Removed the ignore, but this looks like it's helping Three.js communicate with React via their rendering cycle.